backend/game_server/lib/loops/loop_move.ex

```diff
@@ -0,0 +1,26 @@
+defmodule BernacleServer.Loops.LoopMove do
```

Comment:
u could reverse words in loop name so it sounds as proper english.
`MoveLoop` is more than you mean. like loop of some move calculation.
`LoopMove` sounds as somebody going around and can't stop.